DentOS_Framework/DentOsTestbed/configuration/testbed_config/sit/infra1/QUAGGA_DAEMONS

```diff
@@ -0,0 +1,30 @@
+# This file tells the quagga package which daemons to start.
+#
+# Entries are in the format: <daemon>=(yes|no|priority)
+#   0, "no"  = disabled
+#   1, "yes" = highest priority
+#   2 .. 10  = lower priorities
+# Read /usr/share/doc/quagga/README.Debian for details.
+#
+# Sample configurations for these daemons can be found in
+# /usr/share/doc/quagga/examples/.
+#
+# ATTENTION:
+#
+# When activation a daemon at the first time, a config file, even if it is
+# empty, has to be present *and* be owned by the user and group "quagga", else
+# the daemon will not be started by /etc/init.d/quagga. The permissions should
+# be u=rw,g=r,o=.
+# When using "vtysh" such a config file is also needed. It should be owned by
+# group "quaggavty" and set to ug=rw,o= though. Check /etc/pam.d/quagga, too.
+#
+# The watchquagga daemon is always started. Per default in monitoring-only but
+# that can be changed via /etc/quagga/debian.conf.
+#
+zebra=yes
+bgpd=yes
+ospfd=no
+ospf6d=no
+ripd=no
+ripngd=no
+isisd=no
```

Comment:
Please use UNIX line endings.

Reply:
> Thank you. It’s hard to review with one big commit and no commit per change.

this is just 1 "small" change the SIT testbed cable mapping now matches the SIT testbed diagram, all those files were there from forever and were not matching in any way to SIT diagram nor the mapping, each devices comes with10 files there are 7 devices, so probably removed 50 files and added 70

what needs to be reviewed is the diagram of the testbed and cable mapping (the files are just made now to do what the picture shows)